sandy/core/records.py

```diff
@@ -327,9 +327,40 @@ def write_int(x):
 
 
 def line_numbers(length, istart=1):
+    """
+    Line number creator
+
+    Parameters
+    ----------
+    length : 'int'
+        Number of lines.
+    istart : 'int', optional
+        Number from where start counting. The default is 1.
+
+    Returns
+    -------
+    ilines : `list`
+        List containing the number of each line.
+
+    Examples
+    --------
+    >>> np.array(line_numbers(1.0e6)).max()
```

Comment:
these three tests are not enough.
we should at least check for the size...and maybe more

Reply:
Okay, the real thing is that i test with tls files and now len are the same in all the cases